ext/cache/sqlite.rs

```diff
@@ -39,24 +39,35 @@ pub struct SqliteBackedCache {
 impl SqliteBackedCache {
   pub fn new(cache_storage_dir: PathBuf) -> Result<Self, CacheError> {
     {
-      std::fs::create_dir_all(&cache_storage_dir).map_err(|source| {
-        CacheError::CacheStorageDirectory {
-          dir: cache_storage_dir.clone(),
-          source,
-        }
-      })?;
-      let path = cache_storage_dir.join("cache_metadata.db");
-      let connection = rusqlite::Connection::open(&path).unwrap_or_else(|_| {
-        panic!("failed to open cache db at {}", path.display())
-      });
-      // Enable write-ahead-logging mode.
-      let initial_pragmas = "
+      let env_var_val =
+        std::env::var("DENO_CACHE_FORCE_IN_MEMORY").unwrap_or_default();
```

Comment:
I wonder if we should make these enums more of an enum style for better future proofing. Ex. `DENO_CACHE_DB_MODE=memory` or something like that